client/src/App.js

```diff
@@ -13,16 +15,25 @@ export default class App extends Component {
   }
 
   addToSavedList = movie => {
-    const savedList = this.state.savedList;
+    const savedList = [...this.state.savedList];
+    if(this.state.savedList.some( mov => mov.id === movie.id )) return;
     savedList.push(movie);
     this.setState({ savedList });
   };
 
+  removeFromSavedList = movie => {
+    const savedList = [...this.state.savedList];
+    savedList.pop(movie);
+    this.setState({ savedList });
```

Comment:
How could you make this method remove only the movie we clicked on? I made a remove button as well, but did something entirely different. Check out my code to see how I solved it. https://github.com/BrannanC/React-Router-Movies